src/modules/fancyzones/editor/FancyZonesEditor/Models/LayoutModel.cs

```diff
@@ -251,6 +252,37 @@ public static ObservableCollection<LayoutModel> LoadCustomModels()
         private static ObservableCollection<LayoutModel> _customModels = null;
         private static List<string> _deletedCustomModels = new List<string>();
 
+        public static ObservableCollection<string> LoadMonitors()
+        {
+            try
+            {
+                string persistentFancyZoneSettings = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "Microsoft\\PowerToys\\FancyZones\\zones-settings.json");
+                FileStream inputStream = File.Open(persistentFancyZoneSettings, FileMode.Open);
+                JsonDocument jsonObject = JsonDocument.Parse(inputStream, options: default);

```

Comment:
FileStream and JsonDocument implement IDisposable and are leaked if an exception occurs; wrap with using

Reply:
Thanks @riverar for your feedback, I will use the "using" keyword which would handle closing the input stream if it fails